### PR TITLE
build src jar and upload 2.0-dev.1 jars

### PR DESCRIPTION
**Pull request by danrubel**
@devoncarew 
+ build java src jar
+ upload 2.0-dev.1 build as first cut for IDE integration

### CODE REVIEW

File: java/version.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-version=2.0
+version=2.0-dev.1
```
Comment (devoncarew) [inline java/version.txt]: This file is generated from the version in the markdown spec, so we don't want to hand edit it :)

File: java/build.xml [hunk 2]
```diff
@@ -47,5 +48,6 @@
   <target name="dist" depends="jar,test">
     <!-- copy and rename the library -->
     <copy file="build/vm_service_lib.jar" tofile="dist/vm_service_lib-${service.version}.jar"/>
+    <copy file="build/vm_service_lib_src.jar" tofile="dist/vm_service_lib_src-${service.version}.jar"/>
```
Comment (devoncarew) [inline java/build.xml]: Perhaps put a dev version suffix in the init section, and then use it here to build up the file name?

Comment (danrubel) [inline java/build.xml]: Good idea. Done.

File: java/version.txt (hunk 1 above)
Comment (danrubel) [inline java/version.txt]: Good point. Removed from CL.

**Comment by danrubel** [comment]
@devoncarew Updated PR as per suggestions. PTAL

File: java/build.xml [hunk 3]
```diff
@@ -30,6 +32,7 @@
   <target name="jar" depends="compile">
     <copy file="version.txt" todir="build/classes"/>
     <jar destfile="build/vm_service_lib.jar" basedir="build/classes"/>
+    <jar destfile="build/vm_service_lib_src.jar" basedir="src"/>
```
Comment (devoncarew) [inline java/build.xml]: `-src.jar`?

**Comment by devoncarew** [comment]
lgtm

File: java/build.xml (hunk 3 above)
Comment (danrubel) [inline java/build.xml]: Works for me. Done.

**Comment by danrubel** [comment]
merged as 26b63e5da6f25b479e82e6335ad9951105bd89f5

**Pull request closed without merging**